crates/uv-build/src/lib.rs

```diff
@@ -331,6 +333,10 @@ pub struct SourceBuild {
     package_id: String,
     /// Whether we do a regular PEP 517 build or an PEP 660 editable build
     build_kind: BuildKind,
+    /// Modified PATH that contains the [`venv_bin`, `user_path`, `system_path`] variables in that order
```

Comment:
```suggestion
    /// Modified PATH that contains the `venv_bin`, `user_path` and `system_path` variables in that order
```

Rust docstrings are markdown